neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -19,6 +19,7 @@ namespace Neo.Network.P2P
     internal class ProtocolHandler : UntypedActor
     {
         public class SetFilter { public BloomFilter Filter; }
+        public class Flush { }
```

Comment:
How to send messages to `ProtocolHandler`? I think it's not visible to neo-cli.

Reply:
I got it, Erik. I do not know =/
I think that Shargon was planning to send it via P2P.

```cs
            {
                switch (message)
                {
                    case Flush _: FlushHashes(); break;
                }
                return;
            }
```

But this would open doors for attacks, right?
I think that we should just have commands on neo-cli for flushing theses caches.

Type `flush`, then:
1. flush mempool
2. flush protocolhandler
3. flush taskmanager
4. cancel

Reply:
@shargon, what about remove the P2P flush command (which looks a way to attack nodes) and just allow to push this via neo-cli?

Reply:
This will be drpped by ProtocolHandlerMailbox.

Reply:
Will be dropped by ProtocolHandlerMailbox.